### PR TITLE
Commited after implementing login/logout using devise

### PR DESCRIPTION
**Pull request by mdmahmudurrahman**

### CODE REVIEW

File: app/controllers/registrations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class RegistrationsController < Devise::RegistrationsController
+
+  private
+
+  def sign_up_params
+    params.require(:user).permit(:name, :email, :password, :password_confirmation)
+  end
```
Comment (shakil-fg) [inline app/controllers/registrations_controller.rb]: Additional spacing for methods after private

File: config/database.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ common: &common
   pool: 5
   host: localhost
   username: root
-  password: noor1502
+  password: framgia
 
 development:
   <<: *common
```
Comment (shakil-fg) [inline config/database.yml]: remove this file

Restore database.yml.sample.example

Comment (mdmahmudurrahman) [inline config/database.yml]: Restored

File: app/views/devise/registrations/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Update" %>
+  </div>
+<% end %>
+
+<h3>Cancel my account</h3>
+
+<p>Unhappy? <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete %></p>
```
Comment (chuanhtuan) [inline app/views/devise/registrations/edit.html.erb]: i18n

File: app/views/devise/registrations/edit.html.erb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    <%= f.password_field :password_confirmation, autocomplete: "off" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :current_password %> <i>(we need your current password to confirm your changes)</i><br />
+    <%= f.password_field :current_password, autocomplete: "off" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Update" %>
```
Comment (chuanhtuan) [inline app/views/devise/registrations/edit.html.erb]: i18n

File: app/views/devise/registrations/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    <%= f.password_field :password, autocomplete: "off" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "off" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
```
Comment (chuanhtuan) [inline app/views/devise/registrations/new.html.erb]: i18n

File: app/views/devise/shared/_links.html.erb [hunk 6]
```diff
@@ -0,0 +1,25 @@
+<%- if controller_name != 'sessions' %>
+  <%= link_to "Log in", new_session_path(resource_name) %><br />
```
Comment (chuanhtuan) [inline app/views/devise/shared/_links.html.erb]: i18n

File: config/database.yml.example [hunk 7]
```diff
@@ -5,7 +5,7 @@ common: &common
   pool: 5
   host: localhost
   username: root
-  password: noor1502
+  password: framgia
```
Comment (chuanhtuan) [inline config/database.yml.example]: ???

File: app/views/devise/registrations/edit.html.erb (hunk 4 above)
Comment (mdmahmudurrahman) [inline app/views/devise/registrations/edit.html.erb]: Sir, it is from chatwork conversation. You replied to me that, "in this project, skip internationalization for devise, please :)"

File: config/database.yml.example (hunk 7 above)
Comment (mdmahmudurrahman) [inline config/database.yml.example]: This password is for my local DB. We will remove it from the next pull request.

File: app/views/devise/registrations/edit.html.erb (hunk 4 above)
Comment (chuanhtuan) [inline app/views/devise/registrations/edit.html.erb]: OK (bow)

**Pull request merged**